pom.xml

```diff
@@ -227,8 +227,8 @@
         <artifactId>maven-javadoc-plugin</artifactId>
         <configuration>
           <links>
-            <link>https://hc.apache.org/httpcomponents-core-5.0.x/httpcore5/apidocs/</link>
-            <link>https://hc.apache.org/httpcomponents-core-5.0.x/httpcore5-h2/apidocs/</link>
+            <link>https://hc.apache.org/httpcomponents-core-5.2.x/current/httpcore5/apidocs/</link>
+            <link>https://hc.apache.org/httpcomponents-core-5.2.x/current/httpcore5-h2/apidocs/</link>
             <link>${project.url}/httpclient5/apidocs/</link>
```

Comment:
Why is that an external link? This is the javadoc being created through this reactor, isn't it?

Reply:
@kwin No, they are not. HttpCore is a separate project with a separate release cycle.

Reply:
@ok2c I meant line 232 `<link>${project.url}/httpclient5/apidocs/</link>`.